requirements.txt

```diff
@@ -2,6 +2,7 @@ cutlery>=0.0.3,<0.1.0
 spacy>=3.4.0,<3.5.0
 srsly
 torch>=1.12.0
+mypy>=0.990,<0.1000; platform_machine != "aarch64" and python_version >= "3.7"
```

Comment:
Do we need the ARM restriction?

Reply:
The restriction is to avoid using the non-compiled version in thinc or spacy because it is too slow.